ksql-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -132,26 +135,57 @@ public void execute(
       final ConfiguredStatement<InsertValues> statement,
       final KsqlExecutionContext executionContext,
       final ServiceContext serviceContext
+  ) {
+    final InsertValues insertValues = statement.getStatement();
+    final KsqlConfig config = statement.getConfig()
+        .cloneWithPropertyOverwrite(statement.getOverrides());
+
+    final ProducerRecord<byte[], byte[]> record =
+        buildRecord(statement, executionContext, serviceContext, insertValues, config);
+
+    final String tableStreamName = insertValues.getTarget().name();
```

Comment:
nit: Rather than pull out this into a var here, when its only needed in the exception handler, why not just have `createInsertFailedExceptionMessage` take `insertValues`?

Reply:
In the other place that createInsertFailedExceptionMessage is used, the tableStreamName is needed elsewhere in the method. So this minimises the number of calls to insertValues.getTarget().name().

Reply:
But that's not an expensive method.  Better to have succinct & readable code over micro optimisations.